ext/reflection/tests/mixed_type_internal.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Bug #79652: Wrong ReflectionType for sprintf/printf args parameter
+--FILE--
+<?php
+
+$reflectionFunction = new ReflectionFunction("printf");
```

Comment:
I'd suggest testing `function foo(mixed ...$args)` or `function foo(mixed &...$args)` instead. This doesn't have any direct relation to internal vs userland.

Reply:
Oh, I see now what exactly makes the difference between PURE and FULL mask. :)